### ISSUE TITLE
Exporting Postman Collections happens upside down / downside up

### ISSUE THREAD

**Issue by mickey110** [a85/POSTMan-Chrome-Extension#4358]
## App Details:

This happens in Postman - Windows x64 Native App v6.0.10 and v6.0.9, I re-installed until v6.08 and there it's OK.

## Issue Report:

Recently I found that exporting Postman Collections happens upside down / downside up i.e. the resulting .json has last folder/request first, then the request before that, etc..., ending with the first folder/request.

Expected behaviour:  requests/folders are exported in the same order as saved in Postman Collections (GUI).

**Comment by kamalaknn** [comment]
Hi @mickey110 

I just tried exporting a collection from the latest version of the app. I exported in v1, v2 and v2.1 schema formats. The order was correct as represented in the UI.

Could you give us more information to debug this better?

1. What format are you exporting the collection?
2. Does it happen for all collections or only a particular collection?
3. After you import the exported collection in Postman(as a copy) does it show up in the same order?

Please keep in mind that in the v1 schema format the order of `requests` and `folders` does not matter. The actual order of the collection is represented in `order` and `folders_order` array.

**Comment by mickey110** [comment]
1. Collection v2.1 (I always choose recommended).
2 and 3. 
It happened in a particular collection, I did not try other collections at first. In the followup I tried version 5.5.3 up until 6.0.10  to export/import the same particular collection, but I cannot reproduce the issue anymore. 

I suspect that in a past version the order in that particular collection got upside down somehow (an incident that could have many causes), hence the result of the export was upside down but not because of the export.

Anyway, thanks for the quick reply and I think because I cannot reproduce it at the moment we can close the issue.
